config/install/page_manager.page_variant.dashboard-layout_builder-0.yml

```diff
@@ -24,29 +27,28 @@ variant_settings:
           configuration:
             id: 'views_block:message-user_activity'
             label: ''
-            provider: views
             label_display: visible
+            provider: views
+            context_mapping: {  }
             views_label: ''
             items_per_page: none
-            context_mapping: {  }
-          additional: {  }
           weight: 0
-        3310a1ac-c5a7-42c0-a06c-2d172bae4b08:
-          uuid: 3310a1ac-c5a7-42c0-a06c-2d172bae4b08
+          additional: {  }
+        91dbfc09-fade-424e-89e4-189af2e83a72:
+          uuid: 91dbfc09-fade-424e-89e4-189af2e83a72
           region: second
           configuration:
-            id: 'views_block:event_invites-my_events_block'
+            id: 'views_block:event_invites_and_chat_appointments-my_event_invites_and_chat_appointments_block'
```

Comment:
This block should be added by dip_chat_counseling rewrite configs, because it depends in event_invites_and_chat_appointments view that is added in dip_chat_counseling.